src/main/java/jenkins/branch/WorkspaceLocatorImpl.java

```diff
@@ -551,10 +552,15 @@ public void onOnline(Computer c, TaskListener listener) throws IOException, Inte
                                     } catch (IOException x) {
                                         LOGGER.log(Level.WARNING, "could not delete workspace " + child, x);
                                         listener.getLogger().println("could not delete workspace " + child + " , wrong file ownership? Review exception in jenkins log and manually remove the directory");
+                                        failures++;
```

Comment:
I'm considering aborting after a single failure but I'd like to know what others think.

Reply:
I don't think we want to abort. 
We want to try to delete all items not matter what happens with other items.  Why wouldn't we?

Reply:
The more i think about this. I don't think we want to even abort this at all. I'll close this